updates/location_asset_mappings/v2.json

```diff
@@ -90,8 +90,99 @@
+            "location_symbol": "ZKJ"
+        },
+        {
+            "asset": "eip155:56/erc20:0xC0041EF357B183448B235a8Ea73Ce4E4eC8c265F",
+            "location": "kucoin",
+            "location_symbol": "COOKIE"
+        },
+        {
+            "asset": "eip155:1/erc20:0xbe0Ed4138121EcFC5c0E56B40517da27E6c5226B",
+            "location": "kucoin",
+            "location_symbol": "ATH"
+        },
+        {
+            "asset": "eip155:8453/erc20:0x24569d33653c404f90aF10A2b98d6E0030D3d267",
+            "location": "kucoin",
+            "location_symbol": "UNA"
+        },
+        {
+            "asset": "eip155:56/erc20:0x0E7779e698052f8fe56C415C3818FCf89de9aC6D",
+            "location": "kucoin",
+            "location_symbol": "ULTI"
+        },
+        {
+            "asset": "eip155:1/erc20:0x10dea67478c5F8C5E2D90e5E9B26dBe60c54d800",
+            "location": "kucoin",
+            "location_symbol": "TAIKO"
+        },
+        {
+            "asset": "eip155:56/erc20:0x1865dc79a9e4B5751531099057D7Ee801033d268",
+            "location": "kucoin",
+            "location_symbol": "LKI"
+        },
+        {
+            "asset": "eip155:137/erc20:0xaF0dC42725db75AE54f5E8945e71017Bc7ACd27d",
+            "location": "kucoin",
+            "location_symbol": "NEXG"
+        },
+        {
+            "asset": "eip155:1/erc20:0x740df024CE73f589ACD5E8756b377ef8C6558BaB",
+            "location": "kucoin",
+            "location_symbol": "HLG"
+        },
+        {
+            "asset": "eip155:8453/erc20:0x99b2B1A2aDB02B38222ADcD057783D7e5D1FCC7D",
+            "location": "kucoin",
+            "location_symbol": "WLTH"
+        },
+        {
+            "asset": "eip155:1/erc20:0x1337420dED5ADb9980CFc35f8f2B054ea86f8aB1",
+            "location": "kucoin",
+            "location_symbol": "SQD"
+        },
+        {
+            "asset": "eip155:56/erc20:0x0203D275D2A65030889aF45ed91D472be3948B92",
+            "location": "kucoin",
+            "location_symbol": "FURY"
+        },
+        {
+            "asset": "eip155:1/erc20:0xf8428A5a99cb452Ea50B6Ea70b052DaA3dF4934F",
+            "location": "kucoin",
+            "location_symbol": "ZERC"
+        },
+        {
+            "asset": "eip155:1/erc20:0x744030Ad4E6c10fAf5483b62473D88a254d62261",
+            "location": "kucoin",
+            "location_symbol": "NLK"
+        },
+        {
+            "asset": "eip155:1/erc20:0x90685e300A4c4532EFCeFE91202DfE1Dfd572F47",
+            "location": "kucoin",
+            "location_symbol": "CTA"
+        },
+        {
+            "asset": "eip155:137/erc20:0x5F2F8818002dc64753daeDF4A6CB2CcB757CD220",
+            "location": "kucoin",
+            "location_symbol": "WSDM"
         }
       ]
     },
-    "updates": []
+    "updates": [
+      {
+        "asset": "eip155:1/erc20:0xeC5483804e637D45cDe22Fa0869656B64b5AB1ab",
+        "location": "kucoin",
+        "location_symbol": "ACE"
+      }
+    ]
```

Comment:
this updates section is outside of `location_asset_mappings` key, should be inside it